### PR TITLE
LITE-18741 Create parameter functionallity

### PR DESCRIPTION
**Pull request by martinconstante**
Get / List product parameter
Get product configuration parameter

### CODE REVIEW

File: src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    private String name;
+
+    @JsonProperty
+    private String title;
+
+    @JsonProperty
+    private String description;
+
+    @JsonProperty
+    private String type;
```
Comment (rahulmondal) [inline src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java:28]: Should be of type enum with possible values.

File: src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    @JsonProperty
+    private String scope;
```
Comment (rahulmondal) [inline src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java:31]: Should be of type enum with possible values.

File: src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+    @JsonProperty
+    private String phase;
```
Comment (rahulmondal) [inline src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java:34]: Should be of type enum with possible values.

**Review by rahulmondal — CHANGES_REQUESTED** [review]

File: src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java (hunk 1 above)
Comment (martinconstante) [reply src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java]: done

File: src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java (hunk 2 above)
Comment (martinconstante) [reply src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java]: Done

File: src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java (hunk 3 above)
Comment (martinconstante) [reply src/main/java/com/cloudblue/connect/api/models/product/CBCProductParameter.java]: Done

**Review by rahulmondal — APPROVED** [review]

**Pull request merged**